app/codeInsee/lib.ts

```diff
@@ -47,10 +56,15 @@ export const getInseeCodeResults = async (
       const departement = mappedRecord[COLUMN_MAPPING_NAMES.DEPARTEMENT.name];
       const natureJuridique =
         mappedRecord[COLUMN_MAPPING_NAMES.NATURE_JURIDIQUE.name];
+      // natureJuridique is taken into account only if it's a valide value, otherwize we use the general one if it's define
```

Comment:
Petite suggestion de typo:
```suggestion
      // natureJuridique is taken into account only if it's a valid value, otherwise we use the general one if it's defined
```